docs/_sidebar.md

```diff
@@ -189,6 +189,7 @@
   - scripts
     - [README](../scripts/README.md)
   - [DOCS_README](DOCS_README.md)
+  - [Monaco editor migration guide](monaco-editor-migration-guide.md)
```

Comment:
should we consider creating a `packages` folder similar to the `plugins` folder? i think the root level of the docs folder might filling up

Reply:
good point! I will update.